src/waveform/renderers/glwaveformrenderersimplesignal.h

```diff
@@ -3,6 +3,8 @@
 
 #include <QOpenGLFunctions_2_1>
```

Comment:
- Why don't we exclude the whole code including the #include directives?
- Using `#pragma once` would remove one level of conditional compilation

Reply:
QT_NO_OPENGL and QT_OPENGL_ES_2 are defined in qconfig.h or qFeatures.h which are usually included via qglobal.h. 

in this case qglobal.h is included in QOpenGLFunctions_2_1 

This is why the test is done after that. 

I can 
#include <QtCore/qglobal.h>

expicite to make this more obvious.